HPXMLtoOpenStudio/resources/constructions.rb

```diff
@@ -2032,6 +2035,14 @@ def self.create_kiva_slab_foundation(model, int_horiz_r, int_horiz_width, int_ve
       foundation.setExteriorVerticalInsulationDepth(UnitConversions.convert(ext_vert_depth, 'ft', 'm'))
     end
 
+    # Exterior horizontal insulation
+    if (ext_horiz_r > 0) && (ext_horiz_depth > 0) && (ext_horiz_width > 0)
```

Comment:
If ext_horiz_depth is zero, we will ignore the insulation, but it seems like it should be allowed. I will change this.